cmd/root.go

```diff
@@ -48,9 +48,14 @@ func init() {
 	env.Config().BindPFlag("ClusterName", rootCmd.PersistentFlags().Lookup("cluster-name"))
 
 	// Get ssh config values from CLI argument
-	rootCmd.PersistentFlags().StringVarP(&env.Login, "ssh-login", "l", "", "OCP3 ssh login")
-	rootCmd.PersistentFlags().StringVarP(&env.PrivateKey, "ssh-keyfile", "k", "", "OCP3 ssh keyfile path")
-	rootCmd.PersistentFlags().StringVarP(&env.Port, "ssh-port", "p", "", "OCP3 ssh port")
+	rootCmd.PersistentFlags().StringP("ssh-login", "l", "", "OCP3 ssh login")
```

Comment:
Why removing the explicit options for 'ssh-login' and others?
We're going back and forth with that, let's keep both short and long one, unless there is good reason not to.